content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,6 +72,30 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/etcd
+
+Example: `apiserver.latency.k8s.io/etcd: "4.730661757s"`
+
+This annotation indiactes the measure of latency incurred inside the storage layer,
```

Comment:
```suggestion
This annotation indicates the measure of latency incurred inside the storage layer,
```

Reply:
Minor nit - can be fixed in a follow-up PR. Or if @Ritikaa96 , you're willing to push the changes within this week - happy to get this merged ASAP.